### PR TITLE
Mount quiz

### PR DESCRIPTION
**Pull request by Tr4pSt3R**

### CODE REVIEW

**Comment by pmcostello** [comment]
:+1: looks good

File: Gemfile [hunk 1]
```diff
@@ -59,9 +59,9 @@ gem 'feedback', '~> 0.3.1'
 gem 'mortgage_calculator', '~> 1.3.5'
 gem 'payday_loans_intervention', '~> 1.4'
 gem 'pensions_calculator', '~> 1.0.0'
+gem 'quiz', git: 'git@github.com:moneyadviceservice/quiz.git'
```
Comment (benbarnett) [inline Gemfile]: Any reason we're using git rather than the gemserver?

Comment (Tr4pSt3R) [inline Gemfile]: @benbarnett there's another gem in RubyGems so to avoid name clash we used this. We could just as well have defined the source

Comment (benbarnett) [inline Gemfile]: Ah ok, cool

**Comment by benbarnett** [comment]
:+1:

**Pull request merged**